product_docs/docs/transporter/ba_preview/upgrading.mdx

```diff
@@ -1,18 +1,17 @@
 ---
 title: "Upgrading"
-description: Learn how to upgrade EDB Transporter to a more recent version. 
+description: Learn how to upgrade EDB Data Migration Service to a more recent version. 
 ---
 
-Upgrades to Reader are straightforward but involve a temporary halt to any data migration.
+Upgrading the EDB Data Migration Reader is straightforward but involves a temporary halt to any data migration.
```

Comment:
@yidian1997 @Xander-run 
I would like to have more context around upgrades. When specifically would they take place? I don't think we should recommend interrupting a streaming migration for an upgrade, correct? We would rather wait until a migration is completed, then upgrade the reader, and then use the new reader for other migrations? 

I was a bit confused about the `1. Stop the EDB Data Migration Reader.` step below.

Reply:
I'm agree, probably upgrading the reader in the middle of a migration makes not a lot of sense. I think this only makes sense if the reader is involved in a very long migration (weeks) and there is an important change in the reader that needs to be deployed (like a security vulnerability)

Reply:
Stopping the reader is just kill the process